### PR TITLE
[4.0]fixed language of 'insert image' to 'Insert Image'

### PR DESCRIPTION
**Pull request by bahl24**
Signed-off-by: Nitish Bahl <nitishbahl24@gmail.com>

Pull Request for Issue - 'insert image' should have capital first char
1. New article->cms content

### Expected result
screenshot from 2019-02-11 23-52-26
### Actual result
screenshot from 2019-02-11 23-53-55

**Should I add a language def which might help in translation in future**

### CODE REVIEW

**Comment by Bakual** [comment]
The comment behind it is actually the important part. That string should be translatable. So it should go through `Text::_()` and the string being defined in a language file.

**Comment by bahl24** [comment]
> The comment behind it is actually the important part. That string should be translatable. So it should go through `Text::_()` and the string being defined in a language file.

Ok, that's what I meant to say by asking if we need a lang def.
@Bakual Done, kindly see if the string name is according to convention or needs to be modified

File: administrator/language/en-GB/en-GB.plg_editors-xtd_image.ini [hunk 1]
```diff
@@ -6,4 +6,4 @@
 PLG_EDITORS-XTD_IMAGE="Button - Image"
 PLG_IMAGE_BUTTON_IMAGE="Image"
 PLG_IMAGE_XML_DESCRIPTION="Displays a button to insert images into an Article. Displays a popup allowing you to configure an image's properties and upload new image files."
-
+PLG_IMAGE_BUTTON_INSERT = "Insert Image"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_editors-xtd_image.ini]: the lines in this file should be sorted alphabetically

File: plugins/editors-xtd/image/image.php [hunk 2]
```diff
@@ -76,7 +76,7 @@ public function onDisplay($name, $asset, $author)
 				'modalWidth' => '80',
 				'tinyPath'   => $link,
 				'confirmCallback' => 'Joomla.getImage(Joomla.selectedFile, \'' . $name . '\')',
-				'confirmText' => 'insert image' // Needs to be translated
+				'confirmText' => Text::_('PLG_IMAGE_BUTTON_INSERT') // Needs to be translated
```
Comment (brianteeman) [inline plugins/editors-xtd/image/image.php]: You can remove the comment now

**Comment by Bakual** [comment]
> Done, kindly see if the string name is according to convention or needs to be modified

The name is fine, just change what Brian was already pointing out then this PR is perfect. Thanks already! 👍

File: administrator/language/en-GB/en-GB.plg_editors-xtd_image.ini (hunk 1 above)
Comment (infograf768) [reply administrator/language/en-GB/en-GB.plg_editors-xtd_image.ini]: Also, please take off spaces before and after the = sign

**Comment by bahl24** [comment]
@brianteeman @Bakual @infograf768 Changes have been incorporated. Kindly test this

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d6b074c04b4e5df8c2e145a9db10a633e68e8181<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23878">issues.joomla.org/tracker/joomla-cms/23878</a>.</sub>

**Review by Bakual — APPROVED** [review]

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on ddd6014c21b19be88fea01b6adab44ed9a8c8592<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23878">issues.joomla.org/tracker/joomla-cms/23878</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23878">issues.joomla.org/tracker/joomla-cms/23878</a>.</sub>

**Comment by Bakual** [comment]
Merged, thanks for the contribution!

**Pull request merged**